openapi-cli/src/test/resources/generators/service/swagger/requestBody/url_form_encode.yaml

```diff
@@ -0,0 +1,72 @@
+openapi: 3.0.1
+info:
+  title: testInlineRequestBody
+  version: 1.0.0
+paths:
+  /user:
+    post:
+      summary: Post operation for the path /user
+      operationId: inline
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                userName:
+                  description: User Name
+                  type: string
+                userPhone:
+                  description: User Phone Number
+                  type: string
+              required:
+                - userName
+                - userPhone
+      responses:
+        200:
+          description: Successful
+          content:
+            application/json:
+              example: Ok
+  /user02:
+    post:
+      summary: Post operation for the path /user
+      operationId: reference
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/User'
+      responses:
+        200:
+          description: Successful
+          content:
+            application/json:
+              example: Ok
+  /user03:
+    post:
+      summary: Post operation for the path /user
+      operationId: noSchema
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema: {}
```

Comment:
Is this the best way to say the request body can only be key/value pairs ?

Reply:
yeah we can use `type: object`  with  the `additionalProperties`  keyword to specify the type of values in key/value pairs. 
```
schema:
   type: object
   additionalProperties:
        type: string
```
This represents the map<string> with respect to the ballerina language.  [reference](https://swagger.io/docs/specification/data-models/dictionaries/#:~:text=Dictionaries%2C%20HashMaps%20and%20Associative%20Arrays)

Reply:
+1 for this.